bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/handler/AwattarBridgeHandler.java

```diff
@@ -126,7 +126,6 @@ void refreshIfNeeded() {
         if (needRefresh()) {
             refresh();
         }
-        updateStatus(ThingStatus.ONLINE);
```

Comment:
From what I see in the code this is incorrect.
It would overwrite the status send from the `refresh()` method.